test/e2e/cni/e2e_suite_test.go

```diff
@@ -0,0 +1,17 @@
+package cni_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+
+	"github.com/kumahq/kuma/pkg/test"
+	"github.com/kumahq/kuma/test/e2e/cni"
+)
+
+func TestE2E(t *testing.T) {
+	test.RunSpecs(t, "E2E CNI Suite")
+}
+
+var _ = Describe("Taint controller", Label("arm-not-supported"), Label("kind-not-supported"), cni.AppDeploymentWithCniAndTaintController)
+var _ = Describe("Old CNI", Label("arm-not-supported"), Label("kind-not-supported"), cni.AppDeploymentWithCniAndNoTaintController)
```

Comment:
Why is arm not supported for CNI?

Reply:
TBH I don't know, I assumed that since it was disabled [in this PR](https://github.com/kumahq/kuma/pull/4231/files#diff-fc947f87a4c657baeac621981fd1188e95b45c3837667f5d4d08d337bf474664R16) it did not work and we're tracking somewhere else when it will work. Maybe @lukidzi knows?

Reply:
There was no image with `arm64` support. If we released an arm image we should probably use it and remove flag from test.

Reply:
I don't think we manually built it for arm64 and released it so I think for the new CNI we can remove the label.

Reply:
I don't think it's this PR scope to create the arm images for the old CNI (and I'm not sure we want to do this anyway) so I'm closing this conversation. Please open up an issue if we want to do that.